contracts/modules/RecoveryManager.sol

```diff
@@ -192,21 +192,25 @@ contract RecoveryManager is BaseModule, RelayerModule {
         }
     }
 
-    function getRequiredSignatures(BaseWallet _wallet, bytes memory _data) internal view returns (uint256) {
+    function getRequiredSignatures(BaseWallet _wallet, bytes memory _data) internal view returns (uint256, OwnerSignature) {
         bytes4 methodId = functionPrefix(_data);
         if (methodId == EXECUTE_RECOVERY_PREFIX) {
-            return SafeMath.ceil(guardianStorage.guardianCount(_wallet), 2);
+            uint numberOfSignaturesRequired = SafeMath.ceil(guardianStorage.guardianCount(_wallet), 2);
+            return (numberOfSignaturesRequired, OwnerSignature.Disallowed);
         }
         if (methodId == FINALIZE_RECOVERY_PREFIX) {
-            return 0;
+            return (0, OwnerSignature.Disallowed);
```

Comment:
This is a detail, but shouldn't it be cheaper to return the Enum value corresponding to 0, i.e. `OwnerSignature.Required`?

Reply:
I tested this with the following tests but gas cost of each didn't change
`should let anyone finalize the recovery procedure after the recovery period (267ms, 57481 gas)`
`should not let anyone finalize the recovery procedure before the end of the recovery period (207ms, 56306 gas)`
I suppose the memory for the return type is reserved and same whether or not they are zero values.